webrtc/videoroom/config/runtime.exs

```diff
@@ -1,10 +1,76 @@
 import Config
 
+defmodule ConfigParser do
```

Comment:
I guess it could land in another file in `config/` directory?

Reply:
We would have to put it in another `.exs` file and require it here. We only use it here so I don't think it's worth

Reply:
Just thought it would look more clean.